src/v/features/feature_table.h

```diff
@@ -38,6 +38,7 @@ struct feature_table_snapshot;
 /// only used at runtime.  Therefore it is safe to re-use an integer that
 /// has been made available by another feature being retired.
 enum class feature : std::uint64_t {
+    panda_linking_dr = 1ULL << 1U,
```

Comment:
we are recycling old features? is this the first time we are doing that?

if so, i think there should be more pomp and circumstance around this... what are the rules, footguns, implications, etc...